cpacker/MemGPT
# feat: set up initial POC for Web UI

**Please describe the purpose of this pull request.**
Is it to add a new feature? Is it to fix a bug?

A new feature adding a Web-Based UI that allows users to interact with MemGPT outside of the CLI.

**How to test**
How can we test your PR during review? What commands should we run? What outcomes should we expect?

Please give it a try and see if you can get it up and running with the following steps:
1. run `memgpt configure`
2. run `memgpt server`
3. go to `http://127.0.0.1:8283` and see if you can create an agent (prefilled with data that works, need to add endpoints to fetch valid options) and chat with them

**Have you tested this PR?**
Have you tested the latest commit on the PR? If so please provide outputs from your tests.

I have tested it and was able to create an Agent and Chat with the agent

**Related issues or PRs**
Please link any related GitHub [issues](https://github.com/cpacker/MemGPT/issues) or [PRs](https://github.com/cpacker/MemGPT/pulls).

**Is your PR over 500 lines of code?**
If so, please break up your PR into multiple smaller PRs so that we can review them quickly, or provide justification for its length.

Justification is that initial setup was necessary to get up and running. Smaller PRs going forward!

**Additional context**
Add any other context or screenshots about the PR here.
